.github/workflows/regression.yml

```diff
@@ -64,3 +64,10 @@ jobs:
       shell: bash
       env:
         NMLC: nmlc
+
+    # Just check we can successfully build it
```

Comment:
for other projects, we do this in "testing.yml", as there might be other things to test for (like flake, black, ..)

Reply:
out of scope for this PR, I feel :p